x/dogfood/keeper/impl_sdk.go

```diff
@@ -138,9 +138,21 @@ func (k Keeper) Unjail(ctx sdk.Context, addr sdk.ConsAddress) {
 // before unjailing it. If the slashing module's unjail function is never called, this
 // function will never be called either.
 func (k Keeper) Delegation(
-	sdk.Context, sdk.AccAddress, sdk.ValAddress,
+	ctx sdk.Context, delegator sdk.AccAddress, validator sdk.ValAddress,
 ) stakingtypes.DelegationI {
-	panic("unimplemented on this keeper")
+	// This interface is only used for unjail to retrieve the self delegation value,
```

Comment:
Is it necessary to use this interface? Or can you create another function that does this but with a different name?

Reply:
It's an interface expected by the slashing module. Currently, we can't change or use a new function because we directly reference the slashing module of the Cosmos SDK.

Reply:
Should we move the slashing module to Exocore so that we can change this interface?